### ISSUE TITLE
For those having issues with getting their fine tunes to develop/load in the xtts_finetune_webui.py

### ISSUE THREAD

**Issue by MackNcD** [daswer123/xtts-webui#11]
I've found a solution.
Use the collab version to produce your finely tuned model (youtube keyword xtts fine tuning -- very simple collab instructions)
Now once you've downloaded the complete fine-tune model from collab, through google drive, 
take you 2.0.2 base model, save it in a safe dir within the main project file and label it something obvious so you don't lose track of it.

Take your three fine-tune model files -- config, model, and vocab, and replace them where the original base model was.

You can now run fine tuned models, which will produce crisp high quality voices quickly for a specific voice (or two, etc) from the original xtts_webui.py. 

You're welcome!
/closed

**Comment by daswer123** [comment]
Hi it was decided to remove xtts-finetune-webui from this repository, since I implemented a faster version in xtts-webui

![Gradio](https://github.com/daswer123/xtts-webui/assets/22278673/689fa7fd-5ed8-46d8-b5c2-a09eed1c030b)
